charts/engine/templates/engine-job-template-gpu-cm.yaml

```diff
@@ -24,7 +24,9 @@ data:
               envFrom:
                 - configMapRef:
                     name: engine-cm
+{{- if or .Values.additional_pvcs .Values.localDataDirectory }}      
               volumeMounts:
+{{- end}}
```

Comment:
Please also add this change to the other engine job template (the `cpu` one)

Reply:
Done, I did it to `volumes` as well unless do you think we shouldn’t